.env

```diff
@@ -2,7 +2,7 @@ API_KEY="eEvdAEQQzFedHQs5h0H04qevA"
 API_SECRET="AFPPyICgVQHvhudG68N4ak8uWwUerV4pEHPgRfjz3BeX4hPloR"
 ACCESS_TOKEN="4855079537-7whIoJLZTdYV7yvap79mv8vif83RsJqgZ3B9gFi"
 ACCESS_TOKEN_SECRET="W8dg9Ft2lgZzITApLZ2sZhuaXC3X3YfIfgchkA59L3uBp"
-GOOGLE_MAP_API="x"
+GOOGLE_MAP_API="AIzaSyBtAJHtVU0nBNBgXRsdqwZndwAVzTzWMxw"
```

Comment:
I'm concerned that we're still committing the .env file. Is there a way around this?